src/main/java/zeenea/connector/common/LabelIdentifier.java

```diff
@@ -0,0 +1,186 @@
+package zeenea.connector.common;
+
+import java.util.*;
+import org.jetbrains.annotations.NotNull;
+import zeenea.connector.exception.ExceptionUtils;
+
+/**
+ * Represents an ordered list of labels for an item, consisting of a list of identification
+ * properties.
+ *
+ * <p>LabelIdentifier don't have to be unique for each item in the Zeenea Data Catalog.
```

Comment:
Pour reprendre la logique au-dessus, et pour éviter les abréviations dans ce genre de document, je te suggère : 
```suggestion
 * <p>LabelIdentifier do not have to be unique for each item in the data source.
```

Reply:
Corrigé